torch/export/_trace.py

```diff
@@ -1541,13 +1575,15 @@ def forward(self, *args, **kwargs):
                 gm, sig = aot_export(wrapped_mod, args, kwargs=kwargs, **flags)
                 log.debug("Exported program from AOTAutograd:\n%s", gm)
 
-            sig.parameters = pytree.tree_map(_strip_root, sig.parameters)
-            sig.buffers = pytree.tree_map(_strip_root, sig.buffers)
-            sig.inputs_to_buffers = pytree.tree_map(_strip_root, sig.inputs_to_buffers)
-            sig.inputs_to_parameters = pytree.tree_map(
-                _strip_root, sig.inputs_to_parameters
-            )
-            sig.buffers_to_mutate = pytree.tree_map(_strip_root, sig.buffers_to_mutate)
+            if sig is not None:
```

Comment:
Can you assert that only in training path sig can be None and add a TODO probably?